recipe/meta.yaml

```diff
@@ -1,24 +1,20 @@
 {% set name = "dask" %}
-{% set version = "2023.6.0" %}
+{% set version = "2023.11.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 980741966ef4d14c62dfb146f315f57d5eaaa6bedc52866f99687bc6054c2d4b
+  sha256: 06b8f39755d37ff6ef4db422774db1f1d5d6788d33f0628c80861dc6452f878c
 build:
   number: 0
   skip: true # [py<39]
 
 requirements:
   host:
     - python
-    - pip
-    - setuptools
-    - wheel
-    - versioneer 0.28
   run:
     - python
     - bokeh >=2.4.2
```

Comment:
Is cytoolz still needed?

Reply:
not sure, it is mentioned in the documentation as optional dependency, for performance.